packages/react/src/hooks/useKey/useKey.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { act, renderHook } from '@testing-library/react'
+import useKey from './useKey'
+
+const INITIAL_VALUE = 0
+
+describe('useKey', () => {
+  describe('refresh function', () => {
+    it('should update key state', async () => {
+      const { result } = renderHook(() => useKey())
+
+      expect(result.current[0]).toBe(INITIAL_VALUE)
+
+      act(() => {
+        result.current[1]()
+      })
+      expect(result.current[0]).toBe(INITIAL_VALUE + 1)
+    })
+
+    it('should update key state not only one', async () => {
+      const { result } = renderHook(() => useKey())
+
+      expect(result.current[0]).toBe(INITIAL_VALUE)
+
+      act(() => {
+        result.current[1]()
+      })
+      expect(result.current[0]).toBe(INITIAL_VALUE + 1)
+
+      act(() => {
+        result.current[1]()
+      })
+      expect(result.current[0]).toBe(INITIAL_VALUE + 2)
+    })
+  })
+})
```

Comment:
How about this? I remove unnecessary `async` keyword, and unnecessary INITIAL_VALUE

```suggestion
import { act, renderHook } from '@testing-library/react'
import useKey from './useKey'

describe('useKey', () => {
  describe('refresh function', () => {
    it('should increment the key state by 1', () => {
      const { result } = renderHook(() => useKey())
      const initialKey = result.current[0]
      const refresh = result.current[1]
      expect(initialKey).toBe(0)

      act(refresh)
      expect(result.current[0]).toBe(initialKey + 1)

      act(refresh)
      expect(result.current[0]).toBe(initialKey + 2)
    })
  })
})
```

Reply:
Your suggestion was the best.

removed unnecessary `async` keywords.
Unnecessary constant `INITIAL_VALUE` was eliminated.

As you have shown, first declare a constant through result.current. 

In the subsequent comparison expect(result.current[0]) it is also good to approach 


Readability has been greatly improved.